index.html

```diff
@@ -1071,7 +1090,12 @@ <h2>Methods</h2>
             </li>
             <li>
               <p>
-                If the {{RTCIceCandidatePair/local}} and {{RTCIceCandidatePair/remote}} attributes of <var>candidatePair</var> do not match a pairing of {{RTCIceCandidatePairEvent/local}} and {{RTCIceCandidatePairEvent/remote}} respectively sent in {{RTCIceTransport/onicecandidatepairadd}}, [= exception/throw =] a {{NotFoundError}}.
+                If |candidatePair| does not [= candidate pair match | match =] any item in [=this=].{{RTCIceTransport/[[CandidatePairs]]}}, [= exception/throw =] a {{NotFoundError}}.
+              </p>
+            </li>
+            <li>
+              <p>
+                [= list/Remove =] all items that [= candidate pair match | match =] |candidatePair| from [=this=].{{RTCIceTransport/[[CandidatePairs]]}}.
```

Comment:
Now we can eliminate "all" here. E.g.
```suggestion
                [= list/Remove =] the item in
                [=this=].{{RTCIceTransport/[[CandidatePairs]]}} that
                [= candidate pair match | matches =] |candidatePair|.
```

Reply:
The nice thing about "all" is that it implicitly addresses the case where there are no matching items.
We can append ", if any" to the suggestion to keep that effect.